src/plugins/intel_cpu/src/emitters/tpp/x64/jit_tpp_emitter.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2020-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/core/rt_info.hpp>
+
+#include "snippets/lowered/linear_ir.hpp"
+#include "snippets/lowered/expression.hpp"
+
+#include "emitters/plugin/x64/jit_emitter.hpp"
+#include "libxsmm.h"
+
+namespace ov {
+namespace intel_cpu {
+class DebugTppEmitter;
+class TppEmitter : public jit_emitter {
```

Comment:
because all other classes are named in camel 😁 
We discussed this a while ago and decided to name all classes in camel and all variables in snake, so I was just honoring this agreement.
As for `jit_emitter` - it's probably a legacy naming from old times.

Anyway, if it bothering you, let's discuss the naming policy on  the sync 🙂

Reply:
I'm satisfied with your explanation, it doesn't bother me anymore 😄